doc/nrf/app_dev/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_flpr.rst

```diff
@@ -0,0 +1,179 @@
+
+This guide provides instructions on how to build and program the nRF54H20 development kit.
+Whether you are working with single or multi-image builds, the following sections will guide you through the necessary steps.
+
+Depending on the sample, you must program only the application core or both the Fast Lightweight Peripheral Processor (FLPR) and the application core.
+Additionally, the process will differ based on whether you are working with a single-image or multi-image build.
+
+.. note::
+   The following instructions do not include multi-image single-core builds scenario.
+
+Building for the application core only
+**************************************
+
+Building for the application core only follows the default building process for the |NCS|.
+For instructions, see how the :ref:`building` page.
+
+.. _building_nrf54h_app_flpr_core:
+
+Building for the application and FLPR core
+******************************************
+
+Building for both the application and the FLPR cores is different from the default |NCS| procedure.
+Using the FLPR core also requires additional configuration to enable it.
+This section outlines how to build and program for both the application and FLPR core, covering separate builds and sysbuild configurations.
+The FLPR core supports two variants:
+
+* ``nrf54h20dk/nrf54h20/cpuflpr``, where FLPR runs from SRAM, which is the recommended method.
+  To build FLPR image with this variant, the application core image must include the ``nordic-flpr`` :ref:`snippet <app_build_snippets>`.
+
+* ``nrf54h20dk/nrf54h20/cpuflpr/xip``, where FLPR runs from MRAM.
+  To build FLPR image with this variant, the application core image must include the ``nordic-flpr-xip`` snippet.
+
+Standard build
+--------------
+
+This subsection focuses on how to build an application using :ref:`sysbuild <configuration_system_overview_sysbuild>`.
+
+.. note::
+   Currently, the documentation does not cover specific instructions for building an application image that uses sysbuild to incorporate the FLPR core as a sub-image.
+   The only documented scenario is for building FLPR as the main image and the application as a sub-image.
+
+Complete the following steps:
+
+.. tabs::
+
+   .. group-tab:: Using minimal sample for VPR bootstrapping
+
+      This option automatically programs the FLPR core with :ref:`dedicated bootstrapping firmware <vpr_flpr_nrf54h20_initiating>`.
+
+      To build and flash both images, run the following command that performs a :ref:`pristine build <zephyr:west-building>`:
+
+      .. code-block:: console
+
+         west build -p -b nrf54h20dk/nrf54h20/cpuflpr
+         west flash
+
+   .. group-tab:: Using application that supports multi-image builds
+
+      If your application involves creating custom images for both the application core and the FLPR core, make sure to disable the VPR bootstrapping sample.
+      You can do this by disabling the ``SB_CONFIG_VPR_LAUNCHER`` option when building for the FLPR target.
```

Comment:
@gmarull Please check if kconfig match, these were originally mentioned for 54L

Reply:
they should be the same